### PR TITLE
Add `dockerfile_nametags` [minor]

### PR DESCRIPTION
**Pull request by ric-evans**
Allows users to use a local image (from a dockerfile) as the base image for another dockerfile?

tested in https://github.com/icecube/skymap_scanner/pull/235

### CODE REVIEW

File: all-dep-logs-from-dockerfiles.sh [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    $GITHUB_ACTION_PATH/dep-log-from-dockerfile.sh \
+        $fname \
+        "dependencies-from-$(basename $fname).log" \
+        "within the container built from \`$fname\`" \
+        $nametag \
+        $USE_PODMAN
+done
+# 2nd - the rest of the dockerfiles
+for fname in ./Dockerfile*; do
+    if [[ $DOCKERFILE_NAMETAGS == *$(basename $fname):* ]]; then
```
Comment (dsschult) [inline all-dep-logs-from-dockerfiles.sh:35]: Does this actually work if `$DOCKERFILE_NAMETAGS` is a list of tags?

**Review by dsschult — COMMENTED** [review]
I'm unsure if this is actually a good idea, or the images should just be sorted before entering the loop (with the base image automatically being the first to run).

I'll also comment that these scripts are starting to get very bash-heavy, and I'm wondering if moving them to python makes them easier.

**Comment by ric-evans** [comment]
> I'm unsure if this is actually a good idea, or the images should just be sorted before entering the loop (with the base image automatically being the first to run).

The primary issue is tagging. Like in https://github.com/icecube/skymap_scanner/blob/main/Dockerfile_no_cvmfs#L1-L3, we need to know what to call what. If we build plain `Dockerfile` first without the tag, then `Dockerfile_no_cvmfs` pulls from DH

File: all-dep-logs-from-dockerfiles.sh (hunk 1 above)
Comment (ric-evans) [reply all-dep-logs-from-dockerfiles.sh:35]: Yes it does -- the list is just a space-separated string

**Review by dsschult — APPROVED** [review]

**Pull request merged**